### DISCUSSION TITLE
Something like Vercel for PocketBase

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#1871] [Q&A]
Hey
I want to deploy my PocketBase project, just like how I easily deploy my other projects to Vercel. Any (free & easy) solutions? (I know about PocketHost but I'm looking for something better).
Thanks.

**Comment by hungcrush** [comment]
@babakfp You can refer this: https://github.com/pocketbase/pocketbase/discussions/537

really really nice 😬

**Reply by babakfp** [reply]
Cool
Thanks

**Reply by babakfp** [reply]
Hmmm, they are asking for credit card stuff...

**Reply by etmartinkazoo** [reply]
I've been using Fly.io and Pocketbase and it is fantastic

**Comment by Fish1** [comment]
I've been using railway. Automated deployments without needing to create docker files.

Caveat being there is current a bug with golang, and no persitant storage. Both being fixed soon.

Been hosting a sveltekit application with them, and it's amazing. Having my entire stack hosted on railway would be nice.

**Reply by babakfp** [reply]
Thanks. Please let me know when things get fixed.

**Comment by babakfp** [comment]
I used a cheap local hosting service provider instead. (It's not an international service provider).
https://liara.ir
